### ISSUE TITLE
Need to add a check against already pulled papers

### ISSUE THREAD

**Issue by GBruening** [alcantarar/literature_update#18]
Need to create a new csv from RYANDATA.csv that updates after every week with the new articles added so we can compare pulled articles to this list and make sure there are no duplicates. 

Do not update RYANDATA.csv EVER.

**Comment by GBruening** [comment]
Go back to vacation Ryan, i'll probably have this in by the time your back.

**Comment by alcantarar** [comment]
Already made a branch, so stay out! Getting tired of this view:

![IMG_20190515_091232 - Copy](https://user-images.githubusercontent.com/38192529/57798716-a8d18000-774d-11e9-87d2-04873b77d405.jpg)

**Comment by GBruening** [comment]
I'll fix it in my own branch damnit!

**Comment by alcantarar** [comment]
2 steps:

1. Add `compare_papers.csv` to LIterature_Updates folder with the rest of the csv files like it is in `topic-modeling` branch. This is a **copy** of `RYANDATA.csv` with columns rearranged to match the weekly literature updates. This allows for easy appending.

2. Ddd this block of code right after weekly update is written to csv:

````
# Compare to previously searched papers
old_papers = pd.read_csv('Literature_Updates/compare_papers.csv')
new_papers = pd.read_csv(fname)
test = []
for i in new_papers['title']: 
    ind = old_papers['title'].str.contains(i, na = False) #see if all of old titles contain new title, skip nans.
    test.append(bool(sum(ind))) #just keep T/F boolean

test = [not tf for tf in test] #list comprehension to flip booleans (we want to keep papers that DON'T match)
new_papers_filt = new_papers[test] #index using boolean
#append new filtered papers to old 'compared_papers.csv'
with open('Literature_Updates/compare_papers.csv', 'a', encoding = 'utf-8', newline = '') as f:
    new_papers_filt.to_csv(f, header=False, index = False)  
````

**Comment by alcantarar** [comment]
EDIT: wifi is really crappy here, can't push12MB `compare_files.csv` in less than 5 mins. I'll upload the csv file later, which will close this issue.

**Comment by GBruening** [comment]
All fixed in b257b0cfdec46d2ca12cd94ba129839dbe814da5

**Comment by alcantarar** [comment]
I'll hopefully get to the PR review and commit https://github.com/alcantarar/literature_update/commit/b257b0cfdec46d2ca12cd94ba129839dbe814da5 tomorrow, but at least before the weekend/bike trip.
